templates/login.html

```diff
@@ -1,11 +1,106 @@
+<!DOCTYPE html>
 <html>
-    <head>
-        <link rel = "stylesheet" href="{{ url_for('static',filename='reg.css') }}">
-    </head>
-    <body>
-        <h1 style="text-align:center">Login Success</h1>
-        <form>
-            <button type="submit" class="registerbtn" formaction = "/logout"><b>Logout</b></button>
-        </form>
-    </body>
-</html>
\ No newline at end of file
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
```

Comment:
put all these files in a layout file and extend that file, since these links are used frequently.

Reply:
Kept in generictemplate.html file